integration_tests/__tests__/version.test.js

```diff
@@ -4,13 +4,15 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
```

Comment:
Yarn types their tests, so I guess he deals with it ;]
I like to just add `// @flow` pragma to get autocompletion and type hints from my editor